src/App.tsx

```diff
@@ -4,29 +4,28 @@ import logo from "../public/FischLogo.svg";
 
 import styles from "./App.module.css";
 import { Card } from "./components/Card";
-import { useState } from "react";
+import { useRef, useState } from "react";
 
 function App() {
+  const [cardsAmount, setCardsAmount] = useState(1)
   const [cards, setCards] = useState([{}]);
```

Comment:
**nitpick:** Why empty object as initial value? Empty array `[]` would be enough

Reply:
One of the requirements is to display at least one card on the first boot, this is that card. :)

Reply:
But why it does not have any values?

Reply:
What do you mean, no values? The card has values, but it only gets created on lines 18-22, where I create a card component for each "empty object" in the cards array. I'll change its name so it's less confusing.